File: joseffe-barroso/Python/aula04/while/exercise35.py
value = int(input("Informe um valor qualquer: "))

while (value <= 0): 
    value = int(input("Informe um valor positivo: "))

index = 1
while (index < 11):
    result = value * index
    print(f"{value} X {index} = {result}")
    index = index + 1	

File: joseffe-barroso/Java/src/aula04/While/Exercise35.java
package While;

import java.util.Scanner;

public class Exercise35 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		int value;
		
		System.out.print("Informe um valor qualquer: ");
		value = scanner.nextInt();
		
		while (value <= 0) {
			System.out.print("Informe um valor positivo: ");
			value = scanner.nextInt();
		}
		
		int index, result;
		index = 1;
		while (index < 11) {
			result = value * index;
			System.out.printf("\n%d X %d = %d", value, index, result);
			index = index + 1;
		}
		
		scanner.close();
	}

}
